### DISCUSSION TITLE
NextJS v11 appropriate loader for .js.map

### DISCUSSION THREAD

**Discussion by joostmeijles** [vercel/next.js#26478] [Help] [answered]
I am using `chrome-aws-lambda` and with the upgrade from NextJS v10 to v11 I get the following webpack error:
```
...
./node_modules/chrome-aws-lambda/build/puppeteer/lib/Browser.js.map
Module parse failed: Unexpected token (1:10)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
```

Seems that there is no appropriate (Webpack 5) loader configured for .js.map files?

How should that be configured?

See full details for reproduction here: https://github.com/alixaxel/chrome-aws-lambda/issues/224

**Comment by timneutkens** [comment]
This means you're trying to bundle chrome-aws-lambda to the browser which will not work. Can you provide a full reproduction?

**Comment by joostmeijles** [comment, marked as answer]
Found the issue while creating a minimal reproduction case, had `target: serverless` in next.config.js (previously added this for deploying on Netlify I think, currently deploying to Vercel anyway).

Removing `target: serverless` solves the issue.

I guess that, after reading this https://github.com/vercel/next.js/issues/20487#issuecomment-753884085, `target:serverless` is no longer supported.

**Reply by goforbg** [reply]
Is there anything else you did? Or can you link a sample small repo doing the same? I've removed target:"serverless" builds on netlify and local doesn't build on Amplify.
